README.adoc

```diff
@@ -252,8 +254,7 @@ include::finish/src/main/liberty/config/server.xml[]
 ----
 
 MicroProfile Metrics API implementers can provide vendor metrics in the same forms as the base and application metrics do.
-Liberty as a vendor supplies server component metrics when both the
-[hotspot=mpMetrics]`mpMetrics` and [hotspot=monitor]`monitor` features are enabled in the [hotspot]`server.xml` configuration file.
+Open Liberty as a vendor supplies server component metrics when [hotspot=mpMetrics]`mpMetrics` feature is enabled in the [hotspot]`server.xml` configuration file.
```

Comment:
Update to 
"Open Liberty as a vendor supplies server component metrics when **the** `mpMetrics` feature is enabled in the `server.xml` configuration file."